Prüfschritte/de/11.1.3.1e Beschriftung von Formularelementen programmatisch ermittelbar.adoc

```diff
@@ -4,109 +4,65 @@ include::include/attributes.adoc[]
-Formularelementen sind z. B. mithilfe von Überschriften sinnvoll strukturiert.
+Visuell voneinander abgesetzte oder logisch miteinander verbundene Gruppen von Formularelementen sind z. B. mithilfe von Überschriften sinnvoll strukturiert.
 
-Ist bei Gruppen von Formularelementen eine Gruppenbeschriftung für das
-Verständnis der Beschriftung der einzelnen Formularelemente nötig, sollte
-die Verfügbarkeit sichergestellt werden.
+Wenn das Verständnis der Beschriftung einzelner Formularelmente einer Gruppe nicht gegeben ist, so ist eine Gruppenbeschriftung erforderlich.
 
 == Warum wird das geprüft?
 
-Die Verknüpfung von Beschriftungen mit den zugeordneten Eingabefeldern stellt
-sicher, dass der Aufbau einer Software-Ansicht unabhängig von der
-Präsentation festgelegt und zugänglich ist:
+Die Verknüpfung von Beschriftungen mit den zugeordneten Eingabefeldern stellt sicher, dass die Struktur unabhängig von der Präsentation wahrnehmbar und zugänglich ist:
 
-* Der Screenreader liest die Beschriftungen vor, wenn der Benutzer durch die
-  Formularelemente wandert.
-* Das jeweils aktive Label kann deutlich hervorgehoben werden ggf. kann eine
-  Hilfstechnologie beim Antippen / Anklicken des Labels den Fokus auf das
-  zugehörige Formularelement setzen.
+* Der Screenreader liest die Beschriftungen vor, wenn der Benutzer durch die Formularelemente navigiert.
+* Das jeweils aktive Label kann deutlich hervorgehoben werden ggf. kann eine Hilfstechnologie beim Antippen / Anklicken des Labels den Fokus auf das zugehörige Formularelement setzen.
 
-Auch wenn die Beschriftungen der Formularelemente korrekt ausgezeichnet sind so
-kann es sein, dass diese nicht ausreichen und z. B. eine gemeinsame
-Überschrift für das Verständnis notwendig ist.
+Auch wenn die Beschriftungen der Formularelemente korrekt ausgezeichnet sind, so kann es sein, dass diese nicht ausreichen und z. B. eine gemeinsame Gruppenbeschriftung für das Verständnis notwendig ist.
 
 == Wie wird geprüft?
 
-=== Anwendbarkeit des Prüfschritts
+=== 1. Anwendbarkeit des Prüfschritts
 
-Der Prüfschritt ist anwendbar, wenn die Software-Ansicht Formularelemente
-enthält.
+Der Prüfschritt ist anwendbar, wenn die App-Ansicht Formularelemente enthält.
 
-=== Prüfung
+=== 2. Prüfung
 
-Ob die Semantik im Quelltext der Anwendung korrekt gesetzt ist, kann in der
-Regel nicht ohne weiteres festgestellt werden.
-Hier muss dann ein Blackbox-Test z. B. mittels Screenreader durchgeführt
-werden.
-Zu beachten ist, das hier auch gruppierte Bedienelemente ggf. eine
-Gruppenbeschriftung benötigen und diese muss auch vom Screenreader ausgegeben
-werden können.
+Ob die Semantik im Quelltext der Anwendung korrekt gesetzt ist, kann in der Regel nicht ohne weiteres festgestellt werden. Hierfür muss ein Test mittels Screenreader durchgeführt werden. Zu beachten ist, dass gruppierte Bedienelemente ggf. eine Gruppenbeschriftung benötigen und diese dann auch vom Screenreader ausgegeben werden muss.
```

Comment:
Hier wäre noch mal zu prüfen, ob es nativ die Möglichkeit der Gruppenbeschriftung gibt oder nicht. Zumindest gibt es sie ja bei Formularen in Webviews, die ja im Apptest auch mitgeprüft werden, und ich vermute dass hier legend bei Fokussierung der inputs (z.B. wenn man gezielt nur Formularelemente durchläuft) ausgegeben wird. 
Die Alternative wäre ja ein vorangehender Text, z.B: "Wollen Sie über unsere Angebote per Mail informiert werden gefolgt von radio inputs "Ja" und "nein", die wohl auch ausreichen würde?